proof/refine/ARM/Schedule_R.thy

```diff
@@ -2303,19 +2303,6 @@ lemma setReprogramTimer_active_sc_at'[wp]:
   unfolding active_sc_at'_def
   by wpsimp
 
-(* FIXME RT: move these whileM lemmas, prove `whileM_*_inv` using `whileM_wp_gen`. *)
```

Comment:
Very minor, but do we want to keep the second half of this FIXME? I have to admit, that it's not completely clear to me what the intention of it was, so maybe we don't care about it anymore

Reply:
This turned out to be easy. The tests failed for some probably annoying reason, so we may as well update this and rerun the tests